README.md

```diff
@@ -1 +1,76 @@
-# Leaderboard-Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard Project
+
+> In this project I have set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. I have developed a first working version of the app following a wireframe, but without styling -Focus is on functionality. I have consumed the Leaderboard API using JavaScript async and await and added some styling.
+
+
+###### Languages 
+- HTML
+- CSS
+- JAVASCRIPT
+###### Tools  
+- Github
+###### Code Editor
+- VS code
+###### Packages 
+- NPM
+- Node
+- Webpack
+
+## Live Demo (if available)
+
+[View Live Demo Here](https://rn486.github.io/Leaderboard-Project/)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
```

Comment:
- [ ] Actually, you need to remove line 28 and line 29 instead, It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd project_name`, etc 🙃  Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.